acro/record.py

```diff
@@ -170,9 +170,10 @@ def serialize_output(self, path: str = "outputs") -> list[str]:
         # move custom files to the output folder
         if self.output_type == "custom":
             for filename in self.output:
-                if os.path.exists(filename):
-                    shutil.copy(filename, path)
-                    output.append(Path(filename).name)
+                for root, _, files in os.walk(os.getcwd()):
+                    if root != os.path.join(os.getcwd(), path) and filename in files:
+                        shutil.copy(os.path.join(root, filename), path)
+                        output.append(Path(os.path.join(root, filename)).name)
```

Comment:
This wont work if (a) If the file doesn't exist (b) the file is in a folder in a parent directory (c) what if there are multiple subdirectories which have the same filename - it'll just use the first one found? There is also no warning if any of these things happen, it just doesn't make it to the destination folder.

Actually, I think if the file doesn't exist the current `custom_output()` will just pretend to add the file without actually checking whether it exists so we should fix that here too...